### PR TITLE
feat: cli delete command

### PR DESCRIPTION
**Pull request by Vafilor**
**What this PR does**:

Adds a basic delete command.

**Which issue(s) this PR fixes**:

Fixes onepanelio/core#270

**Special notes for your reviewer**:

This does not wait to see if all resources have been deleted, so it may take the cluster a few seconds to delete everything.

Use in conjunction with: https://github.com/onepanelio/manifests/pull/102

### CODE REVIEW

**Comment by aleksandrmelnikov** [comment]
@Vafilor Wanted to bring this up for consideration.
I deployed with CLI, using "default" for namespace.
During clean-up:
```
Unable to delete: error when deleting ".onepanel/kubernetes.yaml": namespaces "default" is forbidden: this namespace may not be deleted
```

Since it wasn't deleted, it's not an issue. Not sure if we should point this out via code, or ignore it via code, or not worry about this error.

**Comment by Vafilor** [comment]
> @Vafilor Wanted to bring this up for consideration.
> I deployed with CLI, using "default" for namespace.
> During clean-up:
> 
> ```
> Unable to delete: error when deleting ".onepanel/kubernetes.yaml": namespaces "default" is forbidden: this namespace may not be deleted
> ```
> 
> Since it wasn't deleted, it's didn't impact the k8s cluster.
> I see we use `application-system` in our manifests, so was this supposed to be cleaned-up as a namespace as well?
> 
> * If so, that error may have stopped `application-system` namespace from being deleted.
> 
> Not sure if we should point this out via code, or ignore it via code, or not worry about this error.

@aleksandrmelnikov That's a good point, I didn't consider that case. 
It'd be easiest if we didn't allow the default namespace of course. But that might be inconvenient.

What do you think @rushtehrani? Should we not allow default for now, just rework logic so default is allowed?

**Comment by rushtehrani** [comment]
> What do you think @rushtehrani? Should we not allow default for now, just rework logic so default is allowed?

@Vafilor I am good with disallowing `default` and updating CLI and manifest default and comments accordingly.

**Comment by Vafilor** [comment]
> > What do you think @rushtehrani? Should we not allow default for now, just rework logic so default is allowed?
> 
> @Vafilor I am good with disallowing `default` and updating CLI and manifest default and comments accordingly.

Sounds good. How about `sample` for the new "default" namespace?

**Comment by rushtehrani** [comment]
> Sounds good. How about `sample` for the new "default" namespace?

I think something like `my-project` would make more sense. Ideally we make it `<namespace>` and show an error if they left it as a placeholder.

**Comment by Vafilor** [comment]
> > Sounds good. How about `sample` for the new "default" namespace?
> 
> I think something like `my-project` would make more sense. Ideally we make it `<namespace>` and show an error if they left it as a placeholder.

`<namespace>` with error detection is doable. I'll add this to this PR.

File: util/status.go [hunk 1]
```diff
@@ -12,6 +12,7 @@ func DeploymentStatus(yamlFile *DynamicYaml) (ready bool, err error) {
 	namespacesToCheck["application-system"] = true
 	namespacesToCheck["onepanel"] = true
 	namespacesToCheck["istio-system"] = true
+	namespacesToCheck["default"] = true
```
Comment (rushtehrani) [inline util/status.go:15]: @Vafilor added `default` to reserved ns list here

**Pull request merged**